geomesa/src/main/scala/geotrellis/spark/io/geomesa/GeoMesaAttributeStore.scala

```diff
@@ -0,0 +1,30 @@
+package geotrellis.spark.io.geomesa
+
+import geotrellis.spark.LayerId
+
+import org.geotools.data.DataStoreFinder
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStore
+
+import scala.collection.JavaConverters._
+
+case class GeoMesaAttributeStore(instanceName: String, zookeepers: String, user: String, password: String, useMock: Boolean = false) extends Serializable {
```

Comment:
Yes thought about it, mb `GeoMesaInstance`? and yes, probably we need to have just a Map of fields, to get connector in runtime and to abstract over backend types